benchmark/primitives/primitives_test.go

```diff
@@ -188,6 +189,59 @@ func BenchmarkMutexWithoutDefer(b *testing.B) {
 	}
 }
 
+func BenchmarkIncrInt64Atomics(b *testing.B) {
+	var c int64
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		atomic.AddInt64(&c, 1)
+	}
+	b.StopTimer()
+	if c != int64(b.N) {
+		b.Fatal("error")
+	}
+}
+
+func BenchmarkIncrInt64Mutex(b *testing.B) {
```

Comment:
This is basically identical to `BenchmarkMutex`.  Sure, this is an int64, but the performance will be dominated by the mutex in either case.

Reply:
deleted.